### ISSUE TITLE
Allow Tab to tab to and start typing in a ROLE field in Artist / Draft pages

### ISSUE THREAD

**Issue by jsutan** [vgmdb/tracker#167]
Currently if you press Tab on the field prior to the ROLE field, it will tab to it in the correct order but the field does not immediately accept keyboard input until you click it with your mouse. Can this be changed to allow keyboard input without having to use your mouse?

**Comment by Illidanz** [comment]
You should be able to press ENTER after tabbing on the role field to bring the dropdown up. If that doesn't work, let me know what browser/OS you're on.

But I do agree it should probably just accept any input and focus the search.

**Comment by jsutan** [comment]
Yeah, enter works. Wasn't immediately intuitive but I'll use that from now on until/if this is enhanced.
